content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,61 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+link:https://www.jenkins.io/doc/book/installing/linux/[Linux installations]
+should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+[unordered.stack]
+1) Stop your Jenkins Instance :: 
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+systemctl stop jenkins
+```
```

Comment:
== Upgrade steps on Linux
[[upgrade.linux]]
. Stop your Jenkins instance using the following command:

[source,bash]
```
sudo su -
systemctl stop jenkins
```

This looks to be another set of steps and should be using a numbered list format as well. I had to remove the suggestion formatting so that I could put this in properly. The [anchor](https://docs.asciidoctor.org/asciidoc/latest/attributes/id/#add-additional-anchors-to-a-section) would also need to be updated accordingly, as the `.` delimiter is not interpreted correctly with just a single set of []

Reply:
Because of the same reason of numbers order start from 1 as long as we write any content between [ordered list](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/) we can't use ordered list formatting

Reply:
I was able to find [this information](https://docs.antora.org/antora/latest/asciidoc/description-lists/#complex) and [this information](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/), which is what I had been thinking of when reviewing the lists. There are ways to keep things in line within a list, provided the correct formatting is used. However, I do see the difference between the ordered and description lists in this case. Would it be possible to try this out?

Reply:
Thanks @kmartens27 for your review and time.
I have changed the list style in linux.adoc and windows.adoc, whenever you get time please take a look